Fixes

willkg opened the pull request:
Minor fixes to documentation and infrastructure stuff.

Do the docs look right? If not, then it's probably the case I couldn't figure out how things work.

r?

willkg committed:
Add comment to requirements.txt

Added a comment about why we're using a github hosted version
of django-configurations so that future us know when we can change that
to use a released version.

willkg committed:
Rework README

willkg committed:
Add TeamAdmin

willkg committed:
Cosmetic: flake8-related fixes

willkg commented:
^^^ Two more.

mythmon commented on README.rst:
```diff
@@ -0,0 +1,33 @@
+Run tests
+=========
+
+1. run ``DJANGO_CONFIGURATION=Dev ./manage.py test`` to run Python tests
```
Is there any compelling reason to use RST? I have trouble speaking RST, so if we could stick with Markdown, that would make me happier. But if there is a good reason to go with it, we can do that.

willkg replied:
reST is a lot more flexible than Markdown is, so I do everything in reST. However, if you prefer Markdown, that's fine. I'll switch it to Markdown.

mythmon commented on README.rst:
```diff
@@ -0,0 +1,33 @@
+1. create a virtual environment
+2. run ``pip install -r requirements.txt``
+3. run ``DJANGO_CONFIGURATION=Dev ./manage.py migrate``
+4. run ``npm install``
```
It doesn't matter what order these come in, but I'd put the npm install as step 3, so the first few steps are all about getting dependencies installed.

mythmon commented:
The docs are correct, and your changes look good (modulo the couple of notes about the README). This also makes me want to put a pre-commit linter in here really fast.

willkg committed:
Fix settings

ALLOWED_HOSTS has to be set if DEBUG = False since it affects csrf
checks. Given that, I moved it from the Constants group to the Base
group.

SECRET_KEY was defined twice in Base. I nixed the naughty one.

willkg commented:
^^^ Redid the README in Markdown.

mythmon commented:
Thanks!
